src/Ryujinx.Ava/Assets/Locales/en_US.json

```
@@ -545,6 +545,9 @@
   "SwkbdMinCharacters": "Must be at least {0} characters long",
   "SwkbdMinRangeCharacters": "Must be {0}-{1} characters long",
   "SoftwareKeyboard": "Software Keyboard",
+  "SoftwareKeyboardModeNumbersOnly": "Must be numbers only",
+  "SoftwareKeyboardModeAlphabet": "Must be alphabets only",
+  "SoftwareKeyboardModeASCII": "Must be ASCII text only",
```

```suggestion
  "SoftwareKeyboardModeNumbersOnly": "Must be in numbers (0-9) only",
  "SoftwareKeyboardModeAlphabet": "Must be in English alphabets only",
  "SoftwareKeyboardModeASCII": "Must be in numbers (0-9) and/or English alphabets only",
```

``Must be ASCII text only`` may confuse some users because they may not know about ASCII. While ASCII has the usual punctuation marks such as ``. , ! @ # $ % ^ & * ( )`` and so on (haven't tested which games use these), it also has the control characters (nul, del, etc.) -- see https://en.wikipedia.org/wiki/ASCII#Character_set. So we need to test which games that accepts those characters and change the ``SoftwareKeyboardModeASCII`` message accordingly so that users can understand easily.

---

This should also be changed for Gtk

---

This can also be address separately imo.